Britannia/events/BTWK_storyline_events.txt

```diff
@@ -720,87 +900,286 @@ BTWK_story_event.0105	= { # Guardian Selection Event
 		character = character:3  # Arthur
 		animation = no
 	}
-##########################################
-# Use this to define any truces you want to declare after the council. 
-# Might be a pain, if you want to give all rulers a truce with on another
-#	scope:ntd_brit = {
-#		add_truce_both_ways = {
-#			character = scope:ntd_nord
-#			years = 25
-#			name = TRUCE_SIGNED_PARTITION
-#		}
-#		add_opinion = {
-#			target = scope:ntd_nord
-#			modifier = respect_opinion
-#			opinion = 50
-#		}
-#		reverse_add_opinion = {
-#			target = scope:ntd_nord
-#			modifier = respect_opinion
-#			opinion = 50
-#		}
-#	}
-##########################################
 
+	# NEW UTHER ALLIANCE AFTER MARRIAGE 
 	immediate = {
-		# Does this need an event to alert the player?
-		# NOT HIDDEN ATM
-		character:18 = {
-			remove_trait = restricted_marriage  # Gundleus
-		}
-		character:17 = {
-			remove_trait = restricted_marriage  # Cuneglas
-		}
-		character:12 = {
-			remove_trait = restricted_marriage  # Tristan
-		}
-		# Add alliance between Norwenna's Spouse's Father
-		# CURRENTLY IGNORES THE "NOT" CONDITIONS AND CREATES AN ALLIANCE WITH UTHER TO HIMSELF?
 		if = {
 			limit = {
-				character:19.spouse = { is_landed = no }
-				character:19.spouse.father = { 
-					is_landed = yes
-					NOT = {has_title = title:k_dumnonia}
+				NOT = { 
+					character:3 = {
+						is_spouse_of = character:19
+					}
 				}
 			}
-			event_alliance_effect = {
-				ACTOR = character:1
-				TARGET = character:19.primary_spouse.father
+			# ALLIANCE: UTHER <-> NORWENNA'S SPOUSE'S FATHER IF SPOUSE UNLANDED.
+			if = {
+				limit = {
+					character:19.primary_spouse = { 
+						is_landed = no 
+					}
+					character:19.primary_spouse.father = { 
+						is_alive = yes 
+					}
+				}			
+				event_alliance_effect = {
+					ACTOR = character:1
+					TARGET = character:19.primary_spouse.father
+				}
+			}
+			# ALLIANCE: UTHER <-> NORWENNA'S (LANDED, INDEPENDENT) SPOUSE.
+			else_if = {
+				limit = {
+					character:19.primary_spouse = { 
+						is_landed = yes
+						is_independent_ruler = yes
+					}
+				}
+				event_alliance_effect = {
+					ACTOR = character:1
+					TARGET = character:19.primary_spouse
+				}
+			}
+			# ALLIANCE: UTHER <-> NORWENNA'S SPOUSE'S FATHER IF SPOUSE IS VASSAL UNDER FATHER. 
+			else_if = {
```

Comment:
Could probably combine this into the first if block?

Reply:
It can I think, I'll change it and give it a test